fandogh_cli/fandogh_client/__init__.py

```diff
@@ -77,6 +88,8 @@ def create_version(image_name, version, workspace_path):
 
 def list_versions(image_name):
     response = requests.get(base_images_url + '/' + image_name + '/versions')
+    if response.status_code == 404:
+        return ""
```

Comment:
why empty string here?

Reply:
Empty string signals the caller that there is no version available

Reply:
No version? We return empty list if there is no version.
404 might mean image does not exist